apps/web/src/hooks/useApproveCallback.ts

```diff
@@ -27,20 +28,29 @@ export function useApproveCallback(
   amountToApprove?: CurrencyAmount<Currency>,
   spender?: string,
   options: {
-    addToTransaction
+    addToTransaction?: boolean
     targetAmount?: bigint
+
+    /**
+     * Use paymaster if available.
+     * Enable only if Gas Token Selector is present on the interface.
+     */
+    enablePaymaster?: boolean
   } = {
     addToTransaction: true,
+    enablePaymaster: false,
   },
 ) {
```

Comment:
please set the default value of each filed separately. If we set the default options as a whole, when some of the options are omitted, they'll become undefined by default

Reply:
consider the case where
```typescript
useApproveCallback(amountToApprove, spender)
useApproveCallback(amountToApprove, spender, {
  enablePaymaster: false,
})
```